portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds commentaries element to the page.
+ */
+function showComments() {
+
+  // Response received in the form: {"commentsArray": [{name: "", comment:"", rating:}, {}, {}]}.
+  fetch("/comments").then(response => response.json()).then((commentList) => {
+    const commentsListElement = document.getElementById('comment-field-table');
+    commentsListElement.innerHTML = '';
+
+    for (i in commentList) {
+      commentsListElement.appendChild(createListElement('Name: ' + commentList[i].name));
+      commentsListElement.appendChild(createListElement('Comment text: ' + commentList[i].text));
+      commentsListElement.appendChild(createListElement('Rating: ' + commentList[i].rating));
+      commentsListElement.appendChild(createListElement('\n'));
```

Comment:
Not needed for this project, but it is often a good idea to group elements that relate to one concept in one container. To be more specific, you could do something like create a `<dl>` see (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) element for each item in the commentsList. Then inside you can create a `<dt>` for the field name and `<dd>` for the field value. This approach uses the html tags to provide more context about how the data relates to each other. It won't look the same, but you could modify how it is displayed using css.

Once again, you don't have to change it for this project. This is just informational for future reference.